nekoyume/Assets/_Scripts/UI/Scroller/RecipeScroll.cs

```diff
@@ -295,14 +295,38 @@ public void ShowAsFood(StatType type, bool updateToggle = false)
                 .ToList();
             emptyObjectText.text = L10nManager.Localize("UI_WORKSHOP_EMPTY_CATEGORY");
             emptyObject.SetActive(!items.Any());
-            Show(items, true);
+            Show(items);
+
+            var max = -1;
+            foreach (var item in items)
+            {
+                if (item.Rows.Any(row => row is ConsumableItemRecipeSheet.Row consumableRow
+                    && !IsConsumableLocked(consumableRow)))
+                {
+                    max = Mathf.Max(max, items.IndexOf(item));
+                }
+            }
+            max = max >= 0 ? max : items.Count - 1;
+            JumpTo(max - 1);
```

Comment:
`max - 1` possible to be `-1` when `max == 0`.

```suggestion
            max = max > 0 ? max : items.Count - 1;
            JumpTo(max - 1);
```

Reply:
resolved context
https://planetariumhq.slack.com/archives/CC7BWJYTZ/p1672047289015759